src/sound/sound_source_system.cpp

```diff
@@ -626,7 +626,7 @@ SoundSourceSystem::update(
                                     int randSoundIndex = Game::instance().engine().rng().getInt(0, numOfSounds-1);
                                     std::unordered_map<std::string, std::unique_ptr<Sound>>::iterator soundPointer
                                         = soundSourceComponent->m_sounds.begin();
-                                    for (int i = 0; i < randSoundIndex; ++i)
+                                    for (int i =  0; i < randSoundIndex; ++i)
```

Comment:
actually no, better to remove this since you'll be making another commit when you squash anyway